src/Api/Auth/Controllers/AccountsController.cs

```diff
@@ -264,8 +266,8 @@ public async Task PostKdf([FromBody] KdfRequestModel model)
             throw new UnauthorizedAccessException();
         }
 
```

Comment:
Doesn't look like `PostKdf` has any unit test coverage. Would be good to have the null cases for `model.AuthenticationData` and `model.UnlockData`